### PR TITLE
init fe app with create-react-app and typescript template

### PR DESCRIPTION
**Pull request by Kalpesh3011**
This PR is to initialize the FE app using create-react-app, typescript and tailwindCSS

todo_fe

### CODE REVIEW

File: README.md
Comment (sumeet-baghel) [inline README.md:1]: It'll be better to add this readme inside the `todo-fe` directory with a link on the main readme.

**Review by kunalvarpe — APPROVED** [review]
LGTM

**Review by sumeet-baghel — COMMENTED** [review]
Can you post screenshots of the work done here?

File: README.md
Comment (Kalpesh3011) [reply README.md:1]: done

**Review by sumeet-baghel — APPROVED** [review]

**Pull request merged**